std/process.d

```diff
@@ -3092,7 +3093,7 @@ private:
         import std.array : array;
         LPWSTR lpCommandLine = (to!(wchar[])(line) ~ "\0"w).ptr;
```

Comment:
`to!(WCHAR[])`? Will that work?

Reply:
It'll probably fail to concat with a wchar[] string. I was going to fix dmd to implicitly convert string literals to wchar_t[], but those efforts have hit a brick wall with the bikeshedding over a switch to enable the new behaviors. My goal with this PR anyway is to have it compile regardless of the switch setting.

Reply:
> It'll probably fail to concat with a wchar[] string.

Generally: Argh.

Specifically: you can still change this one to `WCHAR(0)`.

> My goal with this PR anyway is to have it compile regardless of the switch setting.

I think changes to make things pedantically correct are fine whether or not the compiler change happens, as long as it doesn't add more code or uglify the existing one.

Reply:
> I was going to fix dmd to implicitly convert string literals to wchar_t[]

No need it already does https://github.com/dlang/dmd/blob/master/test/compilable/implicitconv.d#L5